apps/daimo-web/src/app/link/[[...slug]]/page.tsx

```diff
@@ -126,6 +127,13 @@ async function loadTitleDesc({ params }: LinkProps): Promise<TitleDesc | null> {
   }
 
   switch (res.link.type) {
+    case "account": {
+      const { account } = res as DaimoAccountStatus;
+      return {
+        title: `${getAccountName(account)} is on Daimo`,
+        description: "Send or recieve payments from them",
```

Comment:
`receive`

Reply:
maybe just `title: getAccountName()` and something like "Get Daimo to send or receive payments"

Reply:
done